documentation/markup.rst

```diff
@@ -10,6 +10,29 @@ This document describes the custom reStructuredText markup introduced by Sphinx
 to support Python documentation and how it should be used.
 
 
+Quick Reference
+===============
+
+This table summarizes which markup should be used for some commonly used
+elements:
+
+================ ========================================= ====================
+Element          Markup                                    See also
+================ ========================================= ====================
+arguments        ``*arg*``                                 :ref:`inline-markup`
+variables        ````var````                               :ref:`inline-markup`
+literals         ````0````, ````[]````                     :ref:`inline-markup`
+True/False/None  ````True````, ````False````, ````None```` :ref:`inline-markup`
+code snippets    ````print('hello world')````              :ref:`inline-markup`
```

Comment:
Since variables, literals and code snippits are all formatted the same and link to the same section, IMO we should just consolidate them; for example (squished a bit to fit within the existing column width constraints):

```suggestion
var/literal/code ````spam````, ````0````, ````1 + 1````    :ref:`inline-markup`
True/False/None  ````True````, ````False````, ````None```` :ref:`inline-markup`
```

Reply:
In light of CAM's `!` hint, I would propose that we instead promote these versions for `None`, `True`, and `False`:

```
:const:`!None`
:const:`!True`
:const:`!False`
```

Reply:
Regarding the consolidation proposed by @CAM-Gerlach, the reasons for splitting those up were:
* having more space for names in the first column
* having more space for additional examples
* making them slightly easier to find

Regarding `True`/`False`/`None`, I think we should stick to ` ``...`` `:
* it's already fairly used consistently throughout the docs
* it's easier to type/remember (especially considering how common they are)

Reply:
Fair enough :)

Reply:
Thanks for elaborating on the reasoning here.

> having more space for names in the first column

Yeah, but if we made the above ``arguments/parameters`` as you suggested, ``variable/literal/code`` would only be one character wider, so the cost would then be minimal.

> having more space for additional examples

Well, I'm not sure we really need more than one of each, since they each represent the three basic classes of ``code``-formatted objects; I'm not sure the additional examples add much additional clarity.

> making them slightly easier to find

Perhaps a little bit, but the fact that it takes up three lines in the table instead of one makes it more to navigate overall and thus everything else harder to find as quickly.

IMO, I still think if we are going to add two more lines, I'd think adding `:class:` and `:meth:` would be a lot more valuable than the marginal advantages repeating variables (which are not even really used much outside of function params/args), literals and code as separate lines would bring. But what do you think @erlend-aasland ?

> Regarding `True`/`False`/`None`, I think we should stick to `` ``...`` ``:

Agreed :+1:

Reply:
> But what do you think @erlend-aasland ?

Sorry, I don't have the bandwidth to participate here for the next few days; I trust your judgement here, CAM.